disvae/encoder.py

```diff
@@ -89,4 +89,4 @@ def forward(self, x):
         mu = self.mu_gen(x)
         log_var = self.log_var_gen(x)  # Log std-dev in paper (bear in mind)
 
-        return mu, log_var
+        return torch.stack([mu, log_var])
```

Comment:
in general in Pytorch we alway leave dimesnion 1 to be batch size. Stacking is totally fine (and more general), but if you want to be clean I'd stack on `dim=-1` (i.e. last dimesnion)

Reply:
Again, nice point, I'll change this